lib/cloud_controller/config_schemas/base/api_schema.rb

```diff
@@ -264,7 +264,8 @@ class ApiSchema < VCAP::Config
             webserver: String, # thin or puma
             optional(:puma) => {
               workers: Integer,
-              max_threads: Integer
+              max_threads: Integer,
+              optional(:max_connections) => Integer
```

Comment:
I find `max_connections` misleading here - I would expect that these are the maximum connections the webserver can handle... It should be `max_db_connections` or even `max_db_connections_per_process`.